views/messages.ejs

```diff
@@ -1,24 +1,73 @@
+<!-- messages.ejs -->
+
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-  <title>messages</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Send Message</title>
+</head>
+<body>
 
-  <meta charset="UTF-8" >
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" >
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+  <h1>Send a Message</h1>
 
-  <link rel="stylesheet" href="/vendor/normalize-4.1.1.css" />
-  <link rel="stylesheet" href="/vendor/border-box.css" />
-  <link rel="stylesheet" href="/styles/main.css" />
-  <link rel="stylesheet" href="/styles/layout.css" />
+  <form id="messageForm">
+    <label for="senderId">Sender ID:</label>
+    <input type="text" id="senderId" name="senderId" required>
 
-  <script src="/vendor/jquery-3.0.0.js"></script>
-  <script defer src="/scripts/app.js"></script>
-</head>
+    <label for="sellerId">reciver ID:</label>
+    <input type="text" id="reciver" name="reciverId" required>
 
-<body>
-  <h1>Heres all your messages</h1>
-</body>
+    <label for="content">Message Content:</label>
+    <textarea id="content" name="content" required></textarea>
+
+    <button type="button" onclick="sendMessage()">Send Message</button>
+  </form>
+
+  <script>
+    function sendMessage() {
+      // Get form data
```

Comment:
Move this function into a separate script file and connect it using the script tag. Then convert it to jquery instead of Vanilla JS